src/azure-cli/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -31,6 +60,162 @@
         az sf application certificate add -g group-name -n cluster1  --secret-identifier 'https://{KeyVault}.vault.azure.net/secrets/{Secret}'
 """
 
+helps['sf application show'] = """
+type: command
+short-summary: Get an application.
+examples:
+  - name: Get application.
+    text: >
+        az sf application show -g testRG -n testCluster --application-name testApp
+"""
+
+helps['sf application list'] = """
+type: command
+short-summary: List applications of a given cluster.
+examples:
+  - name: List applications for a given application type.
+    text: >
+        az sf application list -g testRG -n testCluster
+"""
+
+helps['sf application delete'] = """
+type: command
+short-summary: Delete an application type.
```

Comment:
Delete an application type or application? what is the difference between ` az sf application delete` and `az sf application-type delete`?

Reply:
fixed the description. they're different resources